docs/1_3_shapes.fsx

```diff
@@ -47,8 +47,8 @@ multiple shapes at once.
 
 open Plotly.NET.LayoutObjects
 
-let s1 = Shape.init (StyleParam.ShapeType.Rectangle,2.,4.,3.,4.,Opacity=0.3,Fillcolor="#d3d3d3")
-let s2 = Shape.init (StyleParam.ShapeType.Rectangle,5.,7.,3.,4.,Opacity=0.3,Fillcolor="#d3d3d3")
+let s1 = Shape.init (StyleParam.ShapeType.Rectangle,2.,4.,3.,4.,Opacity=0.3,Fillcolor= Color.fromWebColor "#d3d3d3")
```

Comment:
I would leave out the distinction between "webcolor" and hex and merge them, adding a check if the string starts with a `#`

Reply:
True. It is actually the same. I removed fromHex...

Reply:
Sorry for not being clear, but I would rather name both `fromHex`, this would be the naming web devs would expect when creating colors from hex strings. The name `fromWebColor` implies that things such as `"grey"` or `"aquamarine"` should work as well. So `fromWebColor` should either accept both hex and standard web color strings, or be renamed to `fromHex`